op-plasma/damgr.go

```diff
@@ -240,7 +240,7 @@ func (d *DA) isExpired(bn uint64) bool {
 // trigger a derivation reset.
 func (d *DA) AdvanceL1Origin(ctx context.Context, l1 L1Fetcher, block eth.BlockID) error {
 	// do not repeat for the same origin
-	if block.Number <= d.origin.Number {
```

Comment:
Why was this changed? It seems like we should only advance the origin if the next block is strictly greater.

Reply:
the `AdvanceL1Origin` function is called in `PlasmaDataSource.Next` function, and inside `PlasmaDataSource.Next`, it will call `fetcher.GetInput` -> `state.GetOrTrackChallenge` -> `SetInputCommitment` then push commitment to `activeComms` 

for one block multiple commitments case, `AdvanceL1Origin` will be called multiple times, if use `<=` instead of `<`, the `activeComms` will never get updated